gazebo_plugins/src/gazebo_ros_imu.cpp

```diff
@@ -244,7 +244,7 @@ void GazeboRosIMU::UpdateChild()
     rot = pose.Rot();
 
     // apply rpy offsets
-    rot = this->offset_.Rot()*rot;
+    rot = rot*this->offset_.Rot();
```

Comment:
Would it be possible to condition this based on what version of libsdformat is being used?

Reply:
added version check in fca594d. It assumes that the release will be made in sdformat 6.3.0 and forward ported though. So we'll need to merge and release the [sdformat pull request](https://github.com/osrf/sdformat/pull/500) first